csrankings-t.csv

```diff
@@ -632,7 +632,7 @@ Ting Su 0001,East China Normal University,http://tingsu.github.io,GHPWI7oAAAAJ
 Ting Wang 0006,Pennsylvania State University,https://ist.psu.edu/directory/tbw5359,cwcBTegAAAAJ
 Ting Yu 0001,Qatar Computing Research Institute,http://www.qcri.org.qa/our-people/bio?pid=115&name=Ting_Yu,6filNLkAAAAJ
 Ting Zhong,UESTC,http://www.is.uestc.edu.cn/teachers.do?id=1093,NOSCHOLARPAGE
-Ting Zhu,Univ. of Maryland - Baltimore County,https://www.csee.umbc.edu/~zt/index.html,BXp6PIIAAAAJ
+Ting Zhu,Ohio State University,https://engineering.osu.edu/people/zhu.3445,BXp6PIIAAAAJ

```

Comment:
This is a disambiguation page (https://dblp.org/pid/42/5105.html). Please reach out to DBLP as per instructions.